app/actions/remote/user.ts

```diff
@@ -407,7 +407,7 @@ export const fetchUsersByIds = async (serverUrl: string, userIds: string[], fetc
             return {users: [], existingUsers};
         }
         const users = await client.getProfilesByIds([...new Set(usersToLoad)]);
-        if (!fetchOnly) {
+        if (!fetchOnly && users.length) {
```

Comment:
I guess you added this to remove the warning when no users are returned by the client, not needed by more than ok